docs/install/build.mdx

```diff
@@ -7,22 +7,34 @@ description: |-
 ---
 
 <Tip>
+
 **Building Ghostty from source is not recommended for most users.**
 If you have access to a prebuilt binary or package, you should use that
 instead. See [the binaries and packages page](/docs/install/binary) to
 see if a prebuilt binary or package is available for your platform.
+
 </Tip>
 
-To build Ghostty, you need [Zig 0.14](https://ziglang.org/) installed.
+To build Ghostty, you need [Zig](https://ziglang.org/) installed.
 
 <Important>
-**Zig 0.14 is required.** Ghostty only guarantees that it can build
-against 0.14. Zig is still a fast-moving project so it is likely newer
-versions will not be able to build Ghostty yet. You can find binary
-releases of Zig release builds on the
-[Zig downloads page](https://ziglang.org/download/).
+
+Each version of Ghostty is only guaranteed to build for one **specific** version
+of Zig as it is still a rapidly-evolving project. **Either older or newer
+Zig compilers may not be able to build Ghostty.** If your package manager
+maintains a different version of Zig than what Ghostty requires, you can use
+static binary releases of the Zig compiler on the [Zig downloads page](https://ziglang.org/download/).
+
 </Important>
 
+The Zig version that each Ghostty version requires is as follows:
+
+| Ghostty version | Zig version |
+|-----------------|-------------|
+| 1.0.x           | 0.13.0      |
+| 1.1.x           | 0.13.0      |
+| tip             | 0.14.1      |
```

Comment:
Since this is going into the `tip` branch, we can do `1.2.x` here too to be ready for that release.